docs/main.css

```diff
@@ -0,0 +1,232 @@
+
+.headline {
+    color: #FFFFFF;
+    font-family: "Arial Black";
+    font-size: 72px;
+}
+
+.subheadline {
+    color: #916246;
+    font-family: "Arial Black";
+    font-size: 24px;
+}
+
+.link-container {
+    background-color: rgba(174,174,174,0.65);
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    margin-top: 96px;
+    min-width: 200px;
+    padding: 20px 0px 10px;
+}
+
+.npm-badges {
+    border-radius: 5px;
+    height: 30px;
+    width: 135px;
+    padding-right: 10px;
+}
+
+.gh-btns {
+    border-radius: 5px;
+    padding-left: 5px;
+}
+
+/* Main Content flexbox */
+.main {
+    margin: 0 auto;
+    max-width: 640px;
+    padding: 48px 24px;
+}
+
+/* Info */
+.infoContainer {
+    margin: 0 auto;
+    margin-top: 56px;
+    width: 35%;
+}
+
+#infoTitle {
+    font-family: "Arial Black";
+    font-size: 24px;
+    color: #EE7C5B;
+}
+
+.contents {
+    font-family: "Arial";
+    font-size: 20px;
+    text-align: left;
+    line-height: 24px;
+}
+
+.contents:nth-of-type(1) {
+    margin-top: 32px;
+}
+
+.contents:nth-of-type(2) {
+    margin-top: 16px;
+}
+
+/* Require */
+#requireTitle {
+    color: #EE7C5B;
+    font-family: "Arial Black";
+    font-size: 24px;
+    margin-top: 48px;
+}
+
+#nodelogo {
+    height: 135px;
+    padding: 8px 0px 0px 32px;
+    width: 224px;
+}
+
+#gitlogo {
+    height: 125px;
+    padding: 8px 0px 0px 32px;
+    width: 296px;
+}
+
+.require-links {
+    font-family: "Arial";
+    font-size: 24px;
+}
+
+.main p:nth-of-type(3) {
+    padding-top: 16px;
+}
+
+a {
+    text-decoration: none;
+}
+
+/* Installation */
+#installTitle, #linuxMacTitle, #usageTitle, #sourceTitle {
+    margin-top: 48px;
+    font-family: "Arial Black";
+    font-size: 24px;
+    color: #EE7C5B;
+}
+
+.contents:nth-of-type(5) {
+    margin-top: 16px;
+}
+
+pre {
+    padding: 8px;
+    background-color: #F6F8FA;
+    border-radius: 3px;
+    font-family: 'Courier New', Courier, monospace;
+}
+
+.contents:nth-of-type(6) {
+    margin-top: 16px;
+}
+
+.contents:nth-of-type(7) {
+    margin-top: 8px;
+}
+
+.contents:nth-of-type(8) {
+    margin-top:8px;
+}
+
+/* Linux/Mac */
+.contents:nth-of-type(9) {
+    margin-top: 16px;
+}
+
+/* Usage */
+.contents:nth-of-type(10) {
+    margin-top: 16px;
+}
+
+table {
+    margin-top: 16px;
+    width: 100%;
+    border: 2px solid #979797;
+    border-collapse: collapse;
+}
+
+td {
+    border: 2px solid #979797;
+    font-size: 16px;
+    padding: 8px 0px 8px 8px;
+}
+
+th {
+    border: 2px solid #979797;
+    font-size: 16px;
+    text-align: left;
+    color: #EE7C58;
+    padding: 8px 0px 8px 8px;
+}
+
+.description {
+    font-family: "Arial Black";
+}
+
+.command {
+    font-family: 'Courier New', Courier, monospace;
+}
+
+tr:nth-child(even) {
+    background-color: #D5D5D5;
+}
+
+/* Footer */
+footer {
+    background: radial-gradient(100% 50%, #E0C851 0%, #E93F33 82%);
+    color: #ed9e6d;
+    height: 154px;
+    text-align: center;
+}
+
+#contributing {

```

Comment:
This ID selector and the next one share many properties. Can we just combine them with a class selector?

We want to keep our code [DRY](https://www.codementor.io/joshuaaroke/dry-code-vs-wet-code-89xjwv11w) as much as possible.